apm-collector/apm-collector-core/src/main/java/org/skywalking/apm/collector/core/util/TimeBucketUtils.java

```diff
@@ -88,6 +88,18 @@ public long changeToUTCTimeBucket(long timeBucket) {
         }
     }
 
+    private long addSecondForTimeBucket(String sliceType, long timeBucket, int second) {
```

Comment:
just need a general method to add second for all time bucket type。So this method is public and change the parameter name  sliceType to timeBucketType.